package.json

```diff
@@ -1,12 +1,37 @@
 {
+  "name": "weather-app",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "version": "1.0.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --watch --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
+  },
```

Comment:
- [ ] Kindly install webpack-dev-server in devDependencies for the `npm start` command to execute.